pallets/crowdloan-claim/crowdloan-claim-reward/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+use frame_support::dispatch::{Codec, DispatchResultWithPostInfo};
```

Comment:
Can the content of this file be moved to the pallets/crowdloan-claim-reward ? this package seems a bit orphan.

Reply:
Yes. Good idea.

Reply:
Or wait. You mean as a single crate in the pallets folder or as part of another crate? We could also add it to the new `libs/`directory, as it is not really a pallet...

Reply:
Is it used by both the claim and the reward? If so then yep lib will be fine, otherwise we can define it only where it is used.

Reply:
Yes, it is. Will move it.